drv/hisi_qm_udrv.c

```diff
@@ -75,6 +75,10 @@ static struct hisi_qm_type qm_type[] = {
 		.qm_name	= "hisi_qm_v2",
 		.qm_db_offs	= QM_V2_DOORBELL_OFFSET,
 		.hacc_db	= hacc_db_v2,
+	}, {
+		.qm_name	= "hisi_qm_v3",
+		.qm_db_offs	= QM_V2_DOORBELL_OFFSET,
+		.hacc_db	= hacc_db_v2,
```

Comment:
I think this is wrong, I should think hardware will not change in futurn,  so v2 can run for futurn v3, v4, v5.

Reply:
I need the version handle to compatible interface：